pkg/resource/api.go

```diff
@@ -139,7 +139,11 @@ func (a *APIPatchingApplicator) Apply(ctx context.Context, o runtime.Object, ao
 
 	err := a.client.Get(ctx, types.NamespacedName{Name: m.GetName(), Namespace: m.GetNamespace()}, o)
 	if kerrors.IsNotFound(err) {
-		// TODO(negz): Apply ApplyOptions here too?
+		for _, fn := range ao {
```

Comment:
@muvaf do you anticipate the set of options to always be the same for the `Create` and the `Patch`? It seems like a use case could arise for an option that is applicable for create but not for patch.

Reply:
Sort of, yes. But I converted the PR back to draft because in case of `NotFound` the object isn't changed, meaning it's still a deep copy of the initial one, hence, wouldn't work for my case where I'd need an empty object if `Create` and full object if `Update`.